expression/integration_test.go

```diff
@@ -556,6 +556,7 @@ func (s *testIntegrationSuite) TestMathBuiltin(c *C) {
 }
 
 func (s *testIntegrationSuite) TestStringBuiltin(c *C) {
+	s.ctx.GetSessionVars().SetSystemVar(variable.MaxAllowedPacket, "67108864")
```

Comment:
Perhaps it is better to set it in `mock.NewContext`.

Reply:
`SetSystemVar` may return an error, I still put this line here but added some error check. If we put this into `mock.NewContext`, we'd better change the function returns of `mock.NewContext`?

Reply:
It may bite somebody in the future if he/she write a new TestXXX and forget to set `MaxAllowedPacket`, so IMO it's a better choice to return a error in `mock.NewContext`

Reply:
add `SetSystemVar` setter in `mock.NewContext`, if meets error just panic, although set `MaxAllowedPacket` always returns nil